src/common/options/crimson.yaml.in

```diff
@@ -75,3 +75,8 @@ options:
   level: advanced
   desc: Size in bytes of extents to keep in cache.
   default: 64_M
+- name: seastore_obj_data_write_amplification
+  type: float
+  level: advanced
+  desc: do split extent when write amplification of single _do_transaction_step is greater than this value
```

Comment:
Let's not refer to implementation details in user documentation if we can avoid it.

```
  desc: split extent if ratio of total extent size to write size exceeds this value
```

Reply:
done.